string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,16 @@
+/* exported capitalizeWords */
+function capitalizeWords(string) {
+  var str = string.toLowerCase().split(' ');
+  var arr = [];
+  str.forEach(string => {
+    arr.push(string[0].toUpperCase() + string.slice(1));
+  });
```

Comment:
If you can explain what `str.forEach` is doing, you're approved 👍 . Otherwise, please use a loop.